src/zino/tasks/juniperalarmtask.py

```diff
@@ -16,7 +21,7 @@ async def run(self):
 
         try:
             yellow_alarm_count, red_alarm_count = await self._get_juniper_alarms()
-        except (TypeError, NoSuchNameError):
+        except (EndOfMibViewError, NoSuchInstanceError, NoSuchNameError, NoSuchObjectError, TypeError):
```

Comment:
Might as well utilize the known exception class hierarchies here...

```suggestion
        except (NoSuchNameError, TypeError, VarBindError):
```

Reply:
Good call, didn't think of that!